doc/manual/src/installation/uninstall.md

```diff
@@ -2,11 +2,51 @@
 
 ## Single User
 
-If you have a [single-user installation](./installing-binary.md#single-user-installation) of Nix, uninstall it by running:
+### Alternative:
+ #### Step 1: 
+ Run the script located at `scripts/uninstall-single-user.sh`
```

Comment:
Imagine you're reading this as a first time user that got in trouble with installation or just wants to bail out. Where is that location? What exactly would you have to do to run that script? Currently there is no indication at all, and you "just have to know" that you mean something in the GitHub repository. I insist we avoid this "just have to know" situation at all cost. It got Nix the reputation of being badly documented and inscrutable.

Usually at this point I'd suggest to make it a link to the repo, but that script must match the current Nix version to be sure it will do the right thing. Also it would be quite a few additional steps to run it off the repo. What about we include it as a command in the binary? We could use that opportunity to take inventory of all the file Nix touches, in one central location.

Reply:
That's a really good point, thanks! I definitely agree with you, and I'd like to help Nix be more consistent; swapping this to part of the binary makes sense and I can get to work on that.

Reply:
@fricklerhandwerk Where should I put it in the dir structure? I don't typically develop in C++ but I'm learning as I go 😄

Reply:
We'll discuss among maintainers how to proceed here in general.